warehouse/query-core/src/main/java/datawave/query/jexl/visitors/UnmarkedBoundedRangeDetectionVisitor.java

```diff
@@ -22,26 +34,30 @@ public static boolean findUnmarkedBoundedRanges(JexlNode script) {
     }
 
     @Override
-    public Object visit(ASTReference node, Object data) {
-        // determine if we have a marked range that is not actually a range
-        if (QueryPropertyMarker.findInstance(node).isType(BOUNDED_RANGE)) {
-            if (null != data && !JexlASTHelper.findRange().isRange(node)) {
+    public Object visit(ASTAndNode node, Object data) {
```

Comment:
I'd like a couple of unit tests that hit all the code paths here.

Reply:
Sure thing